Library/src/Systems/Quota/Quota.cpp

```diff
@@ -68,6 +68,18 @@ void FeaturesLimitResult::OnResponse(const csp::services::ApiResponseBase* ApiRe
 	}
 }
 
+int FeaturesLimitResult::ParseErrorCode(const csp::common::String& Value)
+{
+	const int32_t ReturnValue = StringToEQuotaResultFailureReasonIntValue(Value);
```

Comment:
Now that these four overrides look exactly the same, and the base implementation doesn't really do anything atm, we could make this the base implementation and avoid these four overrides entirely.

Reply:
The `ResultBase` implementation of ParseErrorCode is empty on purpose. It is not meant to do anything. There is not a single failure reason that is shared amongst all responses. Also, the enum differs for each result type, and for a good reason. Please leave this as it is.